google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -74,6 +74,7 @@
 
 /** Options for the Cloud Spanner service. */
 public class SpannerOptions extends ServiceOptions<Spanner, SpannerOptions> {
+
```

Comment:
Clarifying question: Were these extra line breaks added by the linting tool?

Reply:
Yes

Reply:
Please recheck if its really the lint tool which generated. I'm not seeing this in other PRs, so we need to understand what's changed.

Reply:
I have removed those extra lines manually and the build has passed. Not sure why running the lint command in my system is generating those lines. Will try to find it out.